packages/contracts-bedrock/src/L2/L1Block.sol

```diff
@@ -73,4 +85,62 @@ contract L1Block is ISemver {
         l1FeeOverhead = _l1FeeOverhead;
         l1FeeScalar = _l1FeeScalar;
     }
+
+    /// @notice Updates the L1 block values for a post-blob activated chain.
+    /// Params are passed in as part of msg.data in order to compress the calldata.
+    /// Params should be passed in in the following order:
+    ///   1. _basefeeScalar      L1 base fee scalar
+    ///   2. _blobBasefeeScalar  L1 blob base fee scalar
+    ///   3. _sequenceNumber     Number of L2 blocks since epoch start.
+    ///   4. _timestamp          L1 timestamp.
+    ///   5. _number             L1 blocknumber.
+    ///   6. _basefee            L1 basefee.
+    ///   7. _blobBasefee        L1 blobBasefee.
+    ///   8. _hash               L1 blockhash.
+    ///   9. _batcherHash        Versioned hash to authenticate batcher by.
+    function setL1BlockValuesEcotone() external {
+        require(msg.sender == DEPOSITOR_ACCOUNT, "L1Block: only the depositor account can set L1 block values");
+
+        uint256 _basefeeScalar;
+        uint256 _blobBasefeeScalar;
+        uint256 _sequenceNumber;
+        uint256 _timestamp;
+        uint256 _number;
+        uint256 _basefee;
+        uint256 _blobBasefee;
+        bytes32 _hash;
+        bytes32 _batcherHash;
+
+        assembly {
+            let offset := 0x4
```

Comment:
I think having an offset for first pass to get passing tests is fine but ideally we don't need to have the intermediate adds and we just hardcode in the values. We should also be able to `sstore` directly in the yul but I am fine with that as a nice to have